spec/requests/api/v1/users/update_spec.rb

```diff
@@ -0,0 +1,78 @@
+RSpec.describe 'PUT api/v1/users', type: :request do
+  let(:user) { create(:user) }
+  let!(:params) { attributes_for(:user, :with_name, :male).extract!(:name, :gender) }
```

Comment:
```suggestion
  let(:params) { attributes_for(:user, :with_name, :male).extract!(:name, :gender) }
```

And this is still complex
```suggestion
  let(:params) { { name: 'Bar Baz', gender: 'male' } }
```